src/models.rs

```diff
@@ -25,4 +25,18 @@ pub struct NewProject {
     pub name: Option<String>,
     pub link: String,
     pub email: String,
-}
\ No newline at end of file
+}
+
+#[derive(Queryable, Debug, Serialize, Deserialize, Insertable, AsChangeset)]
```

Comment:
What if we derived `AsChangeset` on the `Project` struct instead?
Then you should be able to use it and remove the `EditProject` struct…

Reply:
(From https://dzone.com/articles/creating-a-rest-api-in-rust-using-rocket-and-diese )